src/libs/SelectionScraper/index.js

```diff
@@ -43,7 +43,8 @@ const getHTMLOfSelection = () => {
 
         // If clonedSelection has no text content this data has no meaning to us.
         if (clonedSelection.textContent) {
-            let node = null;
+            let parent;
+            let child = clonedSelection;
 
             // If selection starts and ends within same text node we use its parentNode. This is because we can't
```

Comment:
Awesome job @bernhardoj, I didn't get a chance to review the proposal initially, but it's a really good explanation of the fix. 
The code looks good to me. 
Is this comment still true? Should we change it to reflect the changes we're making?

Reply:
I never found a case where the `textContent` is empty. Maybe it will happen when we select an empty html tag, but I don't know how can we select empty html tag. 

Also, I think this PR is not related to the `textContent`, so I assume the comment is still true.